v3/internal/runtime/desktop/api/window.js

```diff
@@ -10,28 +10,38 @@ The electron alternative for Go
 
 /* jshint esversion: 9 */
 
+/**
+ * @typedef {import("./wails").Size} Size
+ * @typedef {import("./wails").Position} Position
+ * @typedef {import("./wails").Screen} Screen
+ */
+
 
 /**
  * The Window API provides methods to interact with the window.
  */
 export const Window = {
     /**
      * Center the window.
+     * @returns {Promise<void>}
      */
     Center: () => void wails.Window.Center(),
     /**
      * Set the window title.
-     * @param title
+     * @param {string} title
+     * @returns {Promise<void>}
```

Comment:
I believe `void` will mean nothing is returned?

Reply:
I believe the function that it calls also returns `Promise<void>`. (https://github.com/jpatters/wails/blob/js-api-types/v3/internal/runtime/desktop/window.js#L65). 

And when I followed it up the chain, it ended up here: https://github.com/jpatters/wails/blob/js-api-types/v3/internal/runtime/desktop/runtime.js#L84

Is this incorrect?

Reply:
Yeah the call to the backend returns a promise but the `void` [here](https://github.com/jpatters/wails/blob/js-api-types/v3/internal/runtime/desktop/window.js#L65) ignores it and returns nothing at all.

Reply:
Ok. Happy to update. What is the desired behaviour? I would have expected a promise.

Reply:
I see. Is there an opportunity for the request to the backend to fail though? Should the promise propagate up?

Reply:
I don't believe so but we could make promise a standard regardless. Just had PRs in the past to remove that when they don't return anything.

Reply:
Yeah. If there's no opportunity for the request from the backend to be rejected then that makes sense for sure. I'll dig through some more code to see what might happen on the backend and come back with a stronger opinion.

Reply:
As you rightly said, those wrappers are out of date so they very well may have changed.

Reply:
So, the backend does return http errors when things go wrong and the response handler does properly call reject when the http code is not in the 200 range. So I would suggest not swallowing those errors and allowing the promise to propagate up to the caller. It will require a few more changes for this to all behave as expected as the method that this calls also swallows the rejection. But minor changes. If this sounds reasonable, I will make the changes in this pr.